tests/php/bootstrap.php

```diff
@@ -5,6 +5,8 @@
  */
 declare(strict_types=1);
 
+error_reporting(E_ALL ^ E_DEPRECATED);
```

Comment:
What's the reason of adding error reporting? Just out of curiousity.

Reply:
I couldn't test the code without it because of deprecation warnings - at least with PHP version 7.4.26:

```
16) MultisiteGlobalMedia\Tests\Unit\AttachmentTest::testAjaxSendAttachmentToEditor
Function ReflectionType::__toString() is deprecated
```

I will remove this line with my next commt.